### PR TITLE
v1.8.4

### PR DESCRIPTION
**Pull request by robertbrignull**
PR to release v1.8.4

## Checklist

- [ ] [CHANGELOG.md](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/CHANGELOG.md) has been updated to incorporate all user visible changes made by this pull request.
- [ ] Issues have been created for any UI or other user-facing changes made by this pull request.
- [ ] _[Maintainers only]_ If this pull request makes user-facing changes that require documentation changes, open a corresponding docs pull request in the [github/codeql](https://github.com/github/codeql/tree/main/docs/codeql/codeql-for-visual-studio-code) repo and add the `ready-for-doc-review` label there.

### CODE REVIEW

File: extensions/ql-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # CodeQL for Visual Studio Code: Changelog
 
-## [UNRELEASED]
+## 1.8.4 - 3 May 2023
 
+- Avoid repeated error messages when unable to monitor a MRVA query. [#2396](https://github.com/github/vscode-codeql/pull/2396)
+- Fix MRVA not displaying the `#select` results set correctly when the query metadata does not exactly match results sets. [#2395](https://github.com/github/vscode-codeql/pull/2395)
+- Adds `codeQL.contextualQueries.disableCache` setting to help library authours to run and debug the find references and find dependencies contetextual queries without relying on the implicit cache. [#2385](https://github.com/github/vscode-codeql/pull/2385)
```
Comment (shati-patel) [inline extensions/ql-vscode/CHANGELOG.md:7]: I think this is internal enough that we don't need to include it in the changelog 🤔 

(But if you'd prefer to keep it, here are some typo fixes 😸 )
```suggestion
- Add `codeQL.contextualQueries.disableCache` setting to help library authors run and debug the "Find references" and "Find dependencies" contextual queries without relying on the implicit cache. [#2385](https://github.com/github/vscode-codeql/pull/2385)
```

**Review by shati-patel — COMMENTED** [review]
Thanks for double-checking the changelog! A couple of minor comments 📝

File: extensions/ql-vscode/CHANGELOG.md (hunk 1 above)
Comment (robertbrignull) [reply extensions/ql-vscode/CHANGELOG.md:7]: I wasn't sure whether to include this or not, so erred on the side of including it. But perhaps we shouldn't bother. It's not included in `package.json` so it isn't advertised to users. I'll remove this line.

**Comment by robertbrignull** [comment]
Thank you @shati-patel for the review 🙇

**Review by shati-patel — APPROVED** [review]
Thanks! 🚢

**Pull request merged**